### DISCUSSION TITLE
Welcome to porefile Discussions!

### DISCUSSION THREAD

**Discussion by iferres** [iferres/long16S#9] [General]
## 👋 Welcome!
  We’re using Discussions as a place to: 
  * Ask questions about `porefile` that are not necessarily issues (bug reports, feature requests, etc).
  * Discuss about long read metaprofiling.
  * Share ideas.
  * Engage with other community members.

**Comment by KFeye** [comment]
Hello!  I accidentally went to the issue page (sorry kinda new) and I have a few questions (also this is super cool).  

So, I am slightly obsessed with the idea of this analysis tool, so I apologize if I ask too many questions. I am trying to run the program (MacOS) and I have been using the barcode kit (96 expansion) and am trying very hard to get to a spot where I have taxa bar plots and diversity analysis.

I demultiplexed and trimmed in guppy. I have 96 folders with fastq.gz files (multiple per barcode). How would I read that data in and do I need to pre-process it any other way (so unzip everything and put it in one fast.gz file?? Also, does it make sense to go to google cloud? Is there a metadata file I need to create?

This is truly great work. Thank you!

**Reply by iferres** [reply]
For newcomers, following this at #11 😄
